src/vireshJJun20/AtmOps.java

```diff
@@ -0,0 +1,61 @@
+package vireshJJun20;
+
+public class AtmOps {
+
+	static int allCredit=0;
+	static int allDebit=0;
+	static int allPrintBal=0;
+	
+	int creditCnt=0;
+	int	debitCnt= 0;
+	int printCnt=0;
+	
+	int creditTr() {
+		allCredit++;
+		creditCnt++;
+		return creditCnt;
+	}
+	
+	int debitTr() {
+		allDebit++;
+		debitCnt++;
+		return debitCnt;
```

Comment:
You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.